### PR TITLE
[INLONG-4153][CI] Add support for first interaction

### PR DESCRIPTION
**Pull request by shink**
fix: #4153

### Motivation

Add support for [first-interaction](https://github.com/actions/first-interaction)

### Modifications

- add a first interaction workflow

### CODE REVIEW

File: .github/workflows/ci_greeting.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Interact with newcomers
+        uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: 'Thanks a lot for opening your first issue with us! 🧡 We''ll get back to you shortly! ⏳'
```
Comment (healchow) [inline .github/workflows/ci_greeting.yml:40]: Please change the `we''ll` to `we'll`, thanks.

Comment (shink) [reply .github/workflows/ci_greeting.yml:40]: @healchow This is an escape character.

Comment (shink) [reply .github/workflows/ci_greeting.yml:40]: The content is correct.
image

File: .github/workflows/ci_greeting.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+  greet:
+    name: Greet
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Interact with newcomers
```
Comment (dockerzhang) [inline .github/workflows/ci_greeting.yml:36]: Interact with Newcomers

Comment (shink) [reply .github/workflows/ci_greeting.yml:36]: @dockerzhang Referring to official documentation, the step name is usually just an initial capital.

File: .github/workflows/ci_greeting.yml (hunk 1 above)
Comment (healchow) [reply .github/workflows/ci_greeting.yml:40]: Okay.

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: .github/workflows/ci_greeting.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: 'Thanks a lot for opening your first issue with us! 🧡 We''ll get back to you shortly! ⏳'
+          pr-message: 'Thanks a lot for your contribution! 💖 This seems to be your first PR! 🌠 Please be sure to follow our [contribution guidelines](../../CONTRIBUTING.md)! 🎁'
```
Comment (dockerzhang) [inline .github/workflows/ci_greeting.yml:41]:
```suggestion
          pr-message: 'Thanks a lot for your contribution! 💖 This seems to be your first PR! 🌠 Please be sure to follow our [contribution guidelines](https://inlong.apache.org/community/how-to-contribute/)! 🎁'
```

**Pull request merged**